google-cloud-spanner/src/test/java/com/google/cloud/spanner/OpenTelemetrySpanTest.java

```diff
@@ -403,10 +385,9 @@ public void multiUse() {
   }
 
   @Test
-  public void transactionRunner() {
+  public void transactionRunner() throws InterruptedException {
```

Comment:
Why are we adding `throws InterruptedException` ? I don't see anything changing in current tests.

Reply:
This was a miss. Updated